### DISCUSSION TITLE
Microsoft SQL Server support

### DISCUSSION THREAD

**Discussion by opensas** [amacneil/dbmate#370] [Feature Ideas]

**Comment by amacneil** [comment]
No plans, but I'm definitely open to reviewing a PR to add support.

**Comment by jreyesco16** [comment]
Hey why did you close the feature request?

**Reply by amacneil** [reply]
I moved it to a discussion. No plans to add this but if someone wants to do so and has a solution for how it can be tested (locally and in CI) then I am open to it.

**Reply by dossy** [reply]
I don't know how serious people who use SQL Server would be willing to use dbmate to make the juice worth the squeeze, here, but I just stumbled across this, which would make testing locally and in CI possible:

https://learn.microsoft.com/en-us/sql/linux/quickstart-install-connect-docker

> Quickstart: Run SQL Server Linux container images with Docker

> This image consists of SQL Server running on Linux based on Ubuntu 20.04. It can be used with the Docker Engine 1.8+ on Linux.

**Comment by Swoorup** [comment]
Probably also need to support `GO` statement (not the go language 😄 ). 

Some functionalities like `Create View` requires to be executed as the first query.
Tools like SQL server management studio, splits each query separated by `GO` and sends them to SQL Server.
